Python/Tests/DjangoTests/DjangoTemplateParserTests.cs

```diff
@@ -106,7 +108,7 @@ public class DjangoTemplateParserTests {
 
 #region Block parser tests
 
-        [TestMethod, Priority(0)]
+        [TestMethod, Priority(TestExtensions.IMPORTANT_UNIT_TEST)]
```

Comment:
these parser tests should be super fast, we could keep all of them in PRs

Reply:
Changed